pkg/kuberesource/kuberesource.go

```diff
@@ -14,12 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kube
+package kuberesource
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-var PodsGroupResource = schema.GroupResource{Group: "", Resource: "pods"}
-var JobsGroupResource = schema.GroupResource{Group: "batch", Resource: "jobs"}
-var NamespacesGroupResource = schema.GroupResource{Group: "", Resource: "namespaces"}
+var Pods = schema.GroupResource{Group: "", Resource: "pods"}
```

Comment:
formatting nit, but I'd (a) sort by group and (b) declare like:
```
var (
    Pods = ...
    Namespaces = ...
)
```
which gofmt will auto-align nicely.